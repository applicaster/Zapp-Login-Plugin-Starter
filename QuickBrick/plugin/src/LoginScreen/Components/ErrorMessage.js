import React from 'react';
import { View, Text } from 'react-native';

const ErrorMessage = ({ errorValue = '', customStyles = {} }) => (
  <View style={styles.container}>
    <Text style={customStyles.fieldsEmptyErrorStyle}>
      {errorValue}
    </Text>
  </View>
);

const styles = {
  container: {
    marginTop: 52
  }
};

export default ErrorMessage;
