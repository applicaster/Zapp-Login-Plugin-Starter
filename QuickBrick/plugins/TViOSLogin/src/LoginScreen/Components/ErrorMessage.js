import React from 'react';
import { View, Text } from 'react-native';

const ErrorMessage = ({ errorValue = '', customStyles = {} }) => (
  <View style={styles.container}>
    <Text style={customStyles.fieldsEmptyErrorStyle}>
      {errorValue}
    </Text>
  </View>
);

const styles = {
  container: {
    marginTop: '8%'
  }
};

export default ErrorMessage;
