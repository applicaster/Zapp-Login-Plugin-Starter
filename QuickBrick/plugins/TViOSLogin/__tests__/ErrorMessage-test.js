import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from '../src/LoginScreen/Components/ErrorMessage';


it('ErrorMessage renders correctly', () => {
  const defaultProps = {
    errorValue: '',
    customStyles: {}
  };

  const tree = renderer
    .create(<ErrorMessage props={defaultProps} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
