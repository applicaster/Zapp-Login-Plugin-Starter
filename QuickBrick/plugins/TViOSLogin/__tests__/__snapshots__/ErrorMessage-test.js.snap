// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": "8%",
    }
  }
>
  <Text>
    
  </Text>
</View>
`;
